finance.md

```diff
@@ -0,0 +1,70 @@
+# Financial Accounts and Practices
+
+_Note: Not all of the following has been implemented yet, the Finance WG is working on it_
+
+This document outlines finance and accounting practices of Hypha Worker Co-operative, including:
+
+- Listing of accounts at our financial institutions
+- Process for invoicing clients and accepted methods of bill payments
+- Process for payroll and expense reimbursement to Members
+- Tools and procedures for bookkeeping, accounting, and tax remittances
+
+## Organizational structure
```

Comment:
This seems like it should be moved into another section from finance?

Reply:
This was moved to onboarding. That work?